client/packages/system/src/Patient/Insurance/InsuranceModal.tsx

```diff
@@ -127,12 +128,37 @@ export const InsuranceModal: FC = (): ReactElement => {
               })
             }
           />
-          <InsuranceStatusToggle
-            isActive={draft.isActive}
-            onChange={isActive =>
-              updatePatch({
-                isActive,
-              })
+          <InputWithLabelRow
```

Comment:
Didn't think this quite justified its own component now.

Also, have left the two checkboxes, but I'm open to reducingn it to just a single toggle if that seems prefereable?